PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java

```diff
@@ -0,0 +1,62 @@
+package com.microsoft.azure.toolkit.appservice.webapp;
+
+import com.microsoft.azure.management.appservice.JavaVersion;
+import com.microsoft.azure.management.appservice.OperatingSystem;
+import com.microsoft.azure.management.appservice.WebApp;
+import com.microsoft.azuretools.core.mvp.model.webapp.AzureWebAppMvpModel;
+import com.microsoft.azuretools.core.mvp.model.webapp.WebAppSettingModel;
+import com.microsoft.azuretools.telemetrywrapper.*;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+
+import static com.microsoft.azuretools.telemetry.TelemetryConstants.CREATE_WEBAPP;
+import static com.microsoft.azuretools.telemetry.TelemetryConstants.WEBAPP;
+
+public class WebAppService {
+    public static WebAppService instance = new WebAppService();
+
+    public static WebAppService getInstance() {
+        return WebAppService.instance;
+    }
+
+    public WebApp createWebApp(final WebAppConfig config) throws Exception {
+        final WebAppSettingModel settings = convertConfig2Settings(config);
+        settings.setCreatingNew(true);
+        final Map<String, String> properties = settings.getTelemetryProperties(null);
+        final Operation operation = TelemetryManager.createOperation(WEBAPP, CREATE_WEBAPP);
+        try {
+            operation.start();
+            EventUtil.logEvent(EventType.info, operation, properties);
+            return AzureWebAppMvpModel.getInstance().createWebApp(settings);
+        } catch (final Exception e) {
+            EventUtil.logError(operation, ErrorType.userError, e, properties, null);
```

Comment:
why this exception is user error?

Reply:
😢 copied from original `WebAppCreationDialg`